Compiler/script2/cs_message_handler.h

```diff
@@ -23,7 +23,7 @@ namespace AGS
 class MessageHandler
 {
 public:
-    enum Severity
+    enum Sev
```

Comment:
Why replace the name? Severity is not too long, and "Sev" is more difficult to guess.

Reply:
For some reason no matter what I do the Severity will cause issue in the compiler like it's picking the wrong thing there - the build error gives a very weird complain. Specifying the field won't work.

Reply:
I think I found a fix but am using the Apple Clang now, will recheck later with gcc 5.4 in Ubuntu.

Reply:
Is the error only caused by member variable declaration `Severity Sev`, or something else besides that?

If only by variable, then my meaning was that it's better to rename a variable than a type.

Reply:
I removed the renaming. It looks like it worked.